### ISSUE TITLE
A11y rules are updated

### ISSUE THREAD

**Issue by jacobkim9881** [uber/baseweb#3890]
Should A11y rules' url on CONTRIBUTING.md be updated from 3.0 to 4.0(currently it's 3.0 on the documentation)?
## Feature description
On line 42 of CONTRIBUTING.md file of this repository A11y rules' URL is https://dequeuniversity.com/rules/axe/3.0/. But A11y rules are updated from 3.0 to 4.0. If the version should be changed then could I work on this? 
## Current
baseweb/ CONTRIBUTING.md
line 42:
- A11y rules can be found [here](https://dequeuniversity.com/rules/axe/3.0/),
## Expected
- A11y rules can be found [here](https://dequeuniversity.com/rules/axe/4.0/),

**Comment by sandgraham** [comment]
Hi @jacobkim9881 thanks for bringing this up. We are currently using `axe-core@v3.1.2` in our tests so the link is accurate but it seems like we should update to `v4` (both the package and the links). Please feel free to give this a shot- there may be some new issues in the unit tests after updating so let us know if you need any guidance.

**Comment by jacobkim9881** [comment]
Hi, @sandgraham, updated `axe-core` version from 3.1.2 to 4.1.0 on package.json file and updated URL on CONTRIBUTIN.md file too. Hope to check #3936!
